compiler/luci/pass/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ target_link_libraries(luci_pass PRIVATE luci_log)
 target_link_libraries(luci_pass PRIVATE luci_service)
 target_link_libraries(luci_pass PRIVATE luci_logex)
 target_link_libraries(luci_pass PRIVATE luci_profile)
+target_link_libraries(luci_pass PRIVATE mio_tflite_inc)
```

Comment:
Could you please separate this change into another PR?
This is not related to introducing FoldDepthwiseConv2D Pass.

Reply:
Sure, I was planning to un-draft #7581 after mio-tflite PR merge.